src/components/Checklist/Checklist.tsx

```diff
@@ -36,7 +36,6 @@ type ChecklistVertical = ChecklistBase & {
 
 type ChecklistHorizontal = ChecklistBase & {
     direction: ChecklistDirection.Horizontal;
-    columns?: undefined;
```

Comment:
@SamuelAlev Are we sure we don't need the columns anymore?

Reply:
Yup, sonarcloud was complaining so I had to fix it